README.md

```diff
@@ -36,7 +36,7 @@ Armada adheres to the CNCF [Code of Conduct](https://github.com/cncf/foundation/
 
 For an overview of the architecture and design of Armada, and instructions for submitting jobs, see:
 
-- [System overview](./docs/design/README.md)
+- [System diagram](./docs/design/diagrams/relationships/README.md)
```

Comment:
```suggestion
- [System diagram](./docs/design/diagrams/relationships/armada_system.png)
```

I think that is correct but not actually sure where this meant to point too.  

@Sharpz7 any ideas where this is supposed to point too?

Reply:
Hey, yeah this is my mistake. I wanted it to point to here:

https://github.com/armadaproject/armada/blob/master/docs/design/relationships_diagram.md

But the link below it actually points to there, so really this link just needs to be removed rather than replaced.

Reply:
yeah sure . I removed that section.